### DISCUSSION TITLE
Feature Request: Optionally Show Weather Effects on Live Wallpaper

### DISCUSSION THREAD

**Discussion by Saijin-Naib** [breezy-weather/breezy-weather#975] [Ideas]
Discovered Breezy not long ago and love it, but I find I am having trouble reading the very subtle Live Wallpaper to understand current conditions. 

Forest Live Wallpaper polls OpenWeatherMap and will draw precipitation effects, cloud cover, etc. 

Something similar, as an option, maybe even showing wind speed and direction, would be amazing.

**Comment by papjul** [comment]
Duplicate of #876, please share your ideas on this thread instead.
